parlai/agents/language_model/language_model.py

```diff
@@ -444,7 +444,10 @@ def vectorize(self, observations, seq_len, is_training):
         else:
             # here we get valid examples and pad them with zeros
             xs, ys, labels, valid_inds, _, y_lens = PaddingUtils.pad_text(
-                observations, self.dict, self.END_IDX, self.NULL_IDX)
+                observations, self.dict, end_idx=self.END_IDX,
+                null_idx=self.NULL_IDX)
+            if xs is None:
+                return None, None, None, None, None
```

Comment:
I think you are working on an old branch because I have a check for when xs is None in the current version. It's slightly different than just returning all None (returns [None]). So you should get rid of my checks when you merge this.

Reply:
See here: https://github.com/facebookresearch/ParlAI/blob/cdf5adee9b07c675d9eb67456c3b0d9d7eeded0e/parlai/agents/language_model/language_model.py#L448